### PR TITLE
Fix Powershell issue with suffix argument that can not be bound

### PR DESCRIPTION
**Pull request by pavlovic-ivan**
**Intention**
This PR fixes the failing job/step that executes Powershell script, with a `suffix` parameter set to empty string.

**Description**
Issue is fixed with `AllowEmptyString` argument added to the `suffix` parameter. Looking at the workflow itself, `suffix` is not set when workflow is created out of ref tag. So, i am assuming it is ok to pass an empty string. If the `suffix` parameter needs to be set all the time, then the workflow itself has to be changed. I am reffering to https://github.com/pavlovic-ivan/ILGPU/blob/master/.github/workflows/ci.yml#L176

@ljubon @mhadzic @mkomatinovic please review :)

### CODE REVIEW

**Review by ljubon — COMMENTED** [review]
I see it's the same idea as explained [here ](https://www.tutorialspoint.com/explain-allowemptystring-and-allowemptycollection-in-powershell-advanced-function)(since I'm at A1 level with PowerShell 💯 )
@pavlovic-ivan it seems you nailed this one :) let's check workflow

File: .github/workflows/Scripts/UpdateSampleReferences.ps1 [hunk 1]
```diff
@@ -15,6 +15,7 @@ Param (
     [string]$version,
 
     [Parameter(Mandatory=$True, HelpMessage="The ILGPU package suffix")]
+    [AllowEmptyString()]
```
Comment (ljubon) [inline .github/workflows/Scripts/UpdateSampleReferences.ps1:18]: Yep, you nailed it ⚡ @pavlovic-ivan  approved

**Review by ljubon — APPROVED** [review]

**Comment by pavlovic-ivan** [comment]
Thanks :)

Got a working example here: https://github.com/pavlovic-ivan/ILGPU/actions/runs/1758011700

**Comment by mhadzic** [comment]
@pavlovic-ivan looks good to me as well. I can not approve it until you add me as a reviewer - as I can see I can not add myself. :)

**Comment by pavlovic-ivan** [comment]
@mhadzic i've sent you an invite to the repo. after accepting it you should be able to review the PR

**Review by mhadzic — APPROVED** [review]

**Pull request closed without merging**